scripts/base/protocols/redis/spicy-decls.zeek

```diff
@@ -0,0 +1,94 @@
+##! Events and records generated by the Redis analyzer.
+
```

Comment:
If you put this in the `Redis` module then you can skip the `Redis::` prefixes below, which is a bit more idiomatic. I actually didn't know that exports work this way, too!